kraken-std/src/kraken/std/python/settings.py

```diff
@@ -139,6 +140,7 @@ def python_settings(
     build_system: PythonBuildSystem | None = None,
     source_directory: str | Path | None = None,
     tests_directory: str | Path | None = None,
+    additional_directories: list[str | Path] | None = None,
```

Comment:
I'm not a big fan of the name ("additional" is vague and non-descriptive), but I don't have a better suggestion... I'll think about it.

Reply:
Naming is hard! Maybe something more descriptive about how these are directories are just being linted? Something like `lint_enforced_directories`?